### ISSUE TITLE
Petstore - OpenApi - Jdk 11 - Compilation issue

### ISSUE THREAD

**Issue by brainwatch** [networknt/light-example-4j#607]
Hi

Do Oracle JDK 11 works with the latest light-example-4j-petstore-openapi ?

Following is the error I get after trying multiple cleanups and cache clearing also. Kindly let me know should I use JDK 11 as you have announced that very soon you will be completely moving out of Jdk 8. I could not get any similar issue traced online anywhere. Hence bringing up this here.
________________________ Console Log __________________________
[INFO] Compiling 6 source files to /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] error reading /Users/brain/.m2/repository/org/yaml/snakeyaml/1.20/snakeyaml-1.20.jar; zip END header not found
[ERROR] error reading /Users/brain/.m2/repository/com/networknt/dump/2.0.9/dump-2.0.9.jar; zip END header not found
[ERROR] error reading /Users/brain/.m2/repository/com/networknt/sanitizer/2.0.9/sanitizer-2.0.9.jar; zip END header not found
[ERROR] error reading /Users/brain/.m2/repository/com/networknt/openapi-meta/2.0.9/openapi-meta-2.0.9.jar; zip END header not found
[ERROR] error reading /Users/brain/.m2/repository/com/networknt/openapi-security/2.0.9/openapi-security-2.0.9.jar; zip END header not found
[ERROR] error reading /Users/brain/.m2/repository/com/networknt/openapi-validator/2.0.9/openapi-validator-2.0.9.jar; zip END header not found
[ERROR] error reading /Users/brain/.m2/repository/com/networknt/specification/2.0.9/specification-2.0.9.jar; zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/model/Error.java:[1,1] cannot access com.networknt.petstore.model
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdDeleteHandler.java:[1,1] cannot access com.networknt.petstore.handler
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsGetHandler.java:[9,8] cannot access com.networknt.petstore
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsGetHandler.java:[9,40] cannot find symbol
  symbol: class LightHttpHandler
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsGetHandler.java:[12,31] cannot find symbol
  symbol:   class HttpServerExchange
  location: class com.networknt.petstore.handler.PetsGetHandler
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPostHandler.java:[9,8] cannot access com.networknt
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPostHandler.java:[9,41] cannot find symbol
  symbol: class LightHttpHandler
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPostHandler.java:[12,31] cannot find symbol
  symbol:   class HttpServerExchange
  location: class com.networknt.petstore.handler.PetsPostHandler
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/model/Pet.java:[8,8] cannot access com
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/model/Pet.java:[11,13] cannot access java
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[3,21] cannot access com.networknt.handler
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[3,29] package com.networknt.handler does not exist
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[4,8] cannot access io
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[4,10] cannot access io.undertow
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[4,26] import requires canonical name for <any>.server.HttpServerExchange
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[5,19] cannot access io.undertow.util
  zip END header not found
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[5,24] package io.undertow.util does not exist
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[9,45] cannot find symbol
  symbol: class LightHttpHandler
[ERROR] /Users/brain/workcloud/learn/networknt/light-example-4j/rest/openapi/petstore/src/main/java/com/networknt/petstore/handler/PetsPetIdGetHandler.java:[12,31] cannot find symbol
  symbol:   class HttpServerExchange
  location: class com.networknt.petstore.handler.PetsPetIdGetHandler
[INFO] 26 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

**Comment by stevehu** [comment]
I have compiled the release and master branch petstore on my local with JDK 11 and there is no issue. I would recommend removing the local .m2 directory and try it again. It looks like something is wrong with the local dependencies as it shows. 

```
END header not found
```

**Comment by stevehu** [comment]
We got another report for the same issue and I have reproduce the issue on my local by removing the  .m2 repository. The problem is caused by a corrupted maven repository in the pom.xml and it is removed for the next release already. 

Please remove the following line from your pom.xml and it should be OK. We are in a process to update all the examples. Thanks a lot for raising it up. 

```
        <repository>
            <id>codelds</id>
            <url>https://code.lds.org/nexus/content/groups/main-repo</url>
        </repository>                          

```

**Comment by brainwatch** [comment]
Hi

Thanks for the quick revert. The issue was by default the example git project clones release branch and once changed that to master it got resolved.

thanks

**Comment by stevehu** [comment]
The master was fixed weeks ago and the release branch was merged two days ago after the new release of 2.0.10 version. I have removed all reference to the maven repo and removed the dependency on Oracle JDBC driver. Thanks again for raising it up.
